galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.ceda.manager/src/main/java/dev/galasa/cicsts/ceda/internal/CEDAManagerField.java

```diff
@@ -0,0 +1,12 @@
+package dev.galasa.cicsts.ceda.internal;
```

Comment:
This needs to be in the dev.galasa.cicsts.ceda package,  the older managers are wrong.  BND objects to this being in a private package.